server/src/testFixtures/java/io/crate/testing/SQLTransportExecutor.java

```diff
@@ -109,7 +109,7 @@ public class SQLTransportExecutor {
     private static final String SQL_REQUEST_TIMEOUT = "CRATE_TESTS_SQL_REQUEST_TIMEOUT";
 
     public static final TimeValue REQUEST_TIMEOUT = new TimeValue(Long.parseLong(
-        Objects.requireNonNullElse(System.getenv(SQL_REQUEST_TIMEOUT), "10")), TimeUnit.SECONDS);
+        Objects.requireNonNullElse(System.getenv(SQL_REQUEST_TIMEOUT), "10")), TimeUnit.MINUTES);
```

Comment:
Why raise the timeout so much?

Reply:
Debug traces leftovers, thanks for catching!

Reply:
This change is also responsible for the `SequenceConsistencyIT.testPrimaryTermIsIncreasedOnReplicaPromotion` failure. Reverting it makes this test pass again.